File: Status_25_07_2022/pattern10.c
#include <stdio.h>

void pattern(int row)
{
    int i=1,j=0,k=1;
    while(i<=row)
    {
        j = i;
        while(j--)
        {
            if(i%2!=0)
            {
                printf("%d",k);
            }
            else
            {
                printf("*");
            }
        }
        printf("\n");
        if(i++%2!=0)
            k++;
    }
}

int main()
{
    int num = 0;
    printf("Enter the number of rows: ");
    scanf("%d",&num);
    pattern(num);
}

File: Status_25_07_2022/pattern10.py
#!/usr/bin/python3

def pattern(row):
    i,j,k = 1,0,1
    while i <= row:
        j = i
        while j:
            j -=1
            if i%2 ==0:
                print("*",end = "")
            else:
                print(k,end = '')
        print()
        i +=1
        if i%2!= 0:
            k +=1
        
pattern(int(input("Enter the number of rows: ")))
